integration-templates/algolia/algolia-create-contacts.ts

```diff
@@ -0,0 +1,25 @@
+import type { NangoAction, AlgoliaContact, AlgoliaCreateContactInput } from './models';
+
+export default async function runAction(nango: NangoAction, input: AlgoliaCreateContactInput): Promise<AlgoliaContact> {
+    const endpoint = `/1/indexes/contacts`;
+
+    try {
+        const postData = {
```

Comment:
validate these inputs

Reply:
This endpoint doesn't have any required fields. If you pass an empty object as your input, a new objectID is created. Do we still need to validate these fields?

Reply:
We should have at least one identifier go into the system. AN empty object doesn't seem like much use.

Reply:
That works, I will add input name validation.